rdfunit-junit/README.md

```diff
@@ -0,0 +1,40 @@
+RDFUnit - JUnit Integration
+==========
+
+RDFUnit-JUnit Integration provides an interface to JUnit via a dedicated Runner which exposes single RDFUnit Test Cases as Unit-Tests.
+
+### Usage
+```java
+
+@RunWith(RdfUnitJunitRunner.class)
+@Schema(uri = "ontologies/foaf.rdf")
+public static class TestRunner {
+
+    @ControlledVocabulary
```

Comment:
If it is meant to be loaded along with the data (which is what I guess from the code) should be renamed to something like `@AdditionalData` but this can also be constructed directly on the `@InputModel`